app/main.py

```diff
@@ -10,25 +12,47 @@
 from app.api.api_v1 import api_router
 from app.core.config import settings, DevPhase
 from app.core.middleware import AuthenticationMiddleware, AuthBackend, LogMiddleware
-from eduhelx_utils.custom_logger import CustomizeLogger
 from app.core.exceptions import CustomException
 
 import logging
 from pathlib import Path
 
+
+formatter = logging.Formatter("%(asctime)s %(levelname)s %(message)s")
+
+def init_logging():
+    logging.getLogger("uvicorn.access").disabled = True
+    root_logger = logging.getLogger()
+    root_logger.disabled = True
+    for handler in root_logger.handlers[:]:
+        root_logger.removeHandler(handler)
+
+def init_uvicorn_logger(formatter) -> logging.Logger:
+    logger = logging.getLogger("uvicorn")
+    logger.setLevel(logging.INFO)
+    stream_handler = logging.StreamHandler(sys.stdout)
+    stream_handler.setFormatter(formatter)
+    logger.addHandler(stream_handler)
+    return logger
+
+def init_file_logger(formatter) -> logging.Logger | None:
+    file_logger = None
+    if settings.DEV_PHASE == DevPhase.PROD:
+        file_logger = logging.getLogger("file-logger")
+        file_logger.handlers.clear()
+        file_logger.setLevel(logging.DEBUG)
+        file_handler = RotatingFileHandler("logs/debug.log", maxBytes=1024*1024*100, backupCount=4)
```

Comment:
do we lose these log files if we kill the pod?

Reply:
I do think we need associated changes on the chart to store logs into a PVC. This file path should probably be controlled by some environment variable rather than fixed, I'm thinking.

Reply:
yeah that was my thought process, whether we'd lose the logs if the pod dies. It should be stored in a PVC but I wouldn't hold back this PR @Hoid - maybe just make a new jira ticket to track the work of persistent logs storage. We can expand on whether that PVC should hold other logs too, something to tackle as part of that ticket IMO